textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexToTrigram.java

```diff
@@ -0,0 +1,109 @@
+package edu.uci.ics.textdb.dataflow.regexmatch;
+
+import com.google.re2j.PublicParser;
+import com.google.re2j.PublicRE2;
+import com.google.re2j.PublicRegexp;
+import com.google.re2j.PublicSimplify;
+
+/**
+ * This class 
+ * {@link https://swtch.com/~rsc/regexp/regexp4.html}
+ * 
+ * @Author Zuozhi Wang
+ * @Author Shuying Lai
+ * 
+ */
+public class RegexToTrigram {	
```

Comment:
This "RegexToTrigram" class is essentially a static helper function, not a class, right?  If so, can we model it as a util function somewhere instead of a class?